platform-api/src/test/java/io/datacater/core/stream/StreamInspectStringTest.java

```diff
@@ -55,7 +55,10 @@ void testStringDeserializer()
     lastMessageToWaitOn.toCompletableFuture().get(1000, TimeUnit.MILLISECONDS);
 
     Response response =
-        given().pathParam("uuid", uuid.toString()).queryParam("limit", "3").get("/{uuid}/inspect");
+        given()
+            .pathParam("uuid", uuid.toString())
+            .queryParams("limit", "3", "sampleMethod", SampleMethod.UNIFORM)
+            .get("/{uuid}/inspect");
```

Comment:
Is this change due to the new query parameter? If so, do we actually test the uniformity of our retrieval, i.e.:

3 Topics hold 30 Items each, if I set limit to 15, I should get 5 from each topic, right?

Reply:
yes, that's why this test changed. Good idea, i added that test case to [StreamInspectMethodsTests](https://github.com/DataCater/datacater/blob/e5b26109672555a0642d0337db1d9008455b770c/platform-api/src/test/java/io/datacater/core/stream/StreamInspectMethodsTest.java)